.github/workflows/ci-dgraph-tests.yml

```diff
@@ -41,6 +41,6 @@ jobs:
           # clean up docker containers before test execution
           ./t -r
           # run the tests
-          ./t --skip contrib,dgraph,ee,filestore,graphql,present,protos,systest,t,tlstest,tok,upgrade,worker,xidmap 
+          ./t --skip contrib,dgraph,ee,filestore,graphql,protos,systest,t,tlstest,tok,upgrade,worker,xidmap  
```

Comment:
@SashaDgraph you introduced a space at the end of line#44 after `xidmap`, can you fix that up?

Every PR should only have relevant diffs, anything outside should be justified.

Reply:
Fixed it now